HISTORY.rst

```diff
@@ -2,6 +2,21 @@
 History
 =======
 
+2.5.8 (2019-09-13)
+------------------
+Breaking changes:
+
+* Patch selenium to convert `executeScript` to `executeAsyncScript`. This is a workaround for the `executeScript` issue in chromedriver 77.
```

Comment:
This should go under Issued Closed. Breaking Changes is for things that require our users to take some action.